### ISSUE TITLE
Sprint 4 – Sprint Planning(19-12-2016)

### ISSUE THREAD

**Issue by lkc442012** [mewind/RSDSupermarketPOS#21]
Hello everyone! Sprint 4 is started today, and i am the scrum master for this week. I hope that after this week, we are able to come out with the final version of the system. So we left only the last part of the system to be develop, which is report generation. Do you guys have any idea about it?

**Comment by mewind** [comment]
Hmm, we should have several types of reports to be generated for this POS system.

**Comment by juzplayplay** [comment]
Maybe we can have exception report? This report will conclude the least sale product in every month so that the staff or manager of the supermarket can understand better about the customer's behaviors.

**Comment by huylin25** [comment]
Yea, so that they can plan their strategies such as reducing the price to increase the sales of that particular product.

**Comment by lkc442012** [comment]
Okay, thats sound great. Any more suggestion?

**Comment by mewind** [comment]
If we have the exception report, then we should also include the monthly sales report in the system also. This sales report will show all the products that were sold in a month. It is similar to exception report but it display the overall results.

**Comment by lkc442012** [comment]
You are right, monthly sales report should be included in our system to provide a detailed description of each item sold.

**Comment by juzplayplay** [comment]
How about summary report? Since the system allows the user to select their card type when they pay, so we can have a summary report on the card transactions in each month.

**Comment by huylin25** [comment]
So the staff or manager can know which card is used more frequently for each month?

**Comment by juzplayplay** [comment]
Yes, you are right. They can track the customer's behavior on the card type they used.

**Comment by lkc442012** [comment]
Okay. I think it is enough to have 3 types of reports in our POS system. So now let us discuss about the task allocation. Who want to in charge of the sales report?

**Comment by huylin25** [comment]
I think I can do that.

**Comment by lkc442012** [comment]
Okay, huy lin. Then you develop the generation of sales report.

**Comment by juzplayplay** [comment]
Then let me take the others. I may have a better idea on how it should be developed.

**Comment by lkc442012** [comment]
Alright. So let me make a conclusion here: 
1) Sales report - Huy Lin
2) Exception report - Chun Xiong
3) Summary report - Chun Xiong

**Comment by mewind** [comment]
If you need any help just inform me, i will try my best to assist you.

**Comment by huylin25** [comment]
Thank you very much.

**Comment by mewind** [comment]
You are welcome!

**Comment by lkc442012** [comment]
If there is no question, we can dismiss the meeting now. Thank you everyone!

**Comment by juzplayplay** [comment]
Okay.
